camelot/admin/action/list_action.py

```diff
@@ -1105,7 +1103,7 @@ def _get_state(self, model_context, filter_value):
         # Only show clear filter mode if any filters are active
         if len(filter_value):
             modes.extend([Mode('__clear', _('Clear filter'), icon=Icon('minus-circle'))])
-        selected_mode_names = [op + '-' + field for field, (op, *_) in filter_value.items()]
+        selected_mode_names = [op.name + '-' + field for field, (op, *_) in filter_value.items()]
```

Comment:
di natuurlijk nt verkeerd, maar op zich is het hier ook nt meer nodig om er een string van te maken, maar kan het ook een lijst met tuples zijn